tests/test_plots.py

```diff
@@ -246,3 +272,52 @@ def test_plot_ridge_aes_labels_shading(self, backend, datatree_4d, pseudo_dim):
         if pseudo_dim != "__variable__":
             assert all(0 in child["alpha"] for child in pc.aes.children.values())
             assert any(pseudo_dim in child["shade"].dims for child in pc.viz.children.values())
+
+    @pytest.mark.parametrize("group", ("prior", "posterior"))
```

Comment:
No need to parametrize by group. It plays an extremely minor role in the function and it is also being checked in the hypothesis tests.